libraries/shared/mocks/log_extractor.go

```diff
@@ -48,3 +48,7 @@ func (extractor *MockLogExtractor) ExtractLogs(recheckHeaders constants.Transfor
 	// return no unchecked headers error so that extractor hits retry interval when delegator under test
 	return logs.ErrNoUncheckedHeaders
 }
+
+func (extractor *MockLogExtractor) BackFillLogs(endingBlock int64) error {
+	panic("implement me")
```

Comment:
Since you're not going to be actually implementing this, as it's a stub, maybe return nil?

Reply:
I'm a little torn on this - I think if we end up with tested code calling this, then it might be good if the `panic` forces us to revisit it and potentially handle on non-nil error. On the other hand, it does raise the question of whether the code that's calling this now should be tested for both success and error handling 🤔